frontend/src/js/controllers/challengeCreate.js

```diff
@@ -0,0 +1,70 @@
+// TODO Still need to work on this file. NOT COMPLETED YET
+(function() {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('ChallengeCreateCtrl', ChallengeCreateCtrl);
+
+    ChallengeCreateCtrl.$inject = ['utilities', '$state', '$http', '$rootScope'];
+
+    function ChallengeCreateCtrl(utilities, $state, $http, $rootScope) {
+        var vm = this;
+        var userKey = utilities.getData('userKey');
+        vm.wrnMsg = {};
+        vm.isValid = {};
+
+        // function to change password
+        vm.createChallenge = function() {
+            // vm.loginContainer = angular.element('.change-passowrd-card');
+
+            var parameters = {};
+            parameters.url = 'auth/password/change/';
+            parameters.method = 'POST';
+            parameters.data = {
```

Comment:
I had to change the data after integrating the datetimepicker in the view. Can you integrate the datetimepicker so that I can proceed? You can make changes to this branch only.